### DISCUSSION TITLE
Preview of upscale model selected

### DISCUSSION THREAD

**Discussion by happysurf** [upscayl/upscayl#587] [Ideas]
Please add a small result preview of image, selecting an upscale model, otherwise I don't know which model is good for my image.

**Comment by aaronliu0130** [comment]
Just check https://github.com/upscayl/upscayl/blob/main/COMPARISONS.MD . 

If you mean previews for specific images, we'll need to Upscayl before granting a preview. this ain't magic, kid

**Reply by happysurf** [reply]
Yes , I know but Upscayl is fast, therefore is not a problem calculate a preview in a small image details window.

**Reply by aaronliu0130** [reply]
It is not fast. It’s resource-intensive and heavily varies depending on what GPUs we use. Plus the preview would be pretty much the same as the actual result.

**Reply by happysurf** [reply]
OK, but the IA is for modern PC, this is clear, in any case the final result show the entire image, only after open and zoom the image in Photoshop, is possible understand the quality of the selected model but obviously after some tests the user knows which model to use depending on the type of image.

**Reply by aaronliu0130** [reply]
I'm not sure what you mean.

**Reply by happysurf** [reply]
How can you evaluate the effectiveness of a model if you don't enlarge the details of an image?
Looking at the whole image it is impossible to understand.

**Reply by aaronliu0130** [reply]
It looks obvious to me

**Reply by happysurf** [reply]
For that, it's better to figure it out first with a small preview window, what the final image quality will be like.
It would be especially useful for less experienced users.
It's just an idea.
